forms/common-themes/src/main/resources/theme/admin/base/resources/partials/realm-credentials.html

```diff
@@ -46,7 +46,9 @@
                             <input class="form-control disabled" type="text" value="{{p.name|capitalize}}" readonly>
                         </td>
                         <td>
-                            <input class="form-control" ng-model="p.value" type="number" placeholder="No value assigned"  min="1">
+                            <input class="form-control" ng-model="p.value" ng-show="p.name != 'notUsername' "

```

Comment:
"not username" doesn't requires any parameters, so, don't show a field for entering the value if the policy is "not usernmae"